UnityProject/Assets/Scripts/Items/Stackable.cs

```diff
@@ -341,4 +341,13 @@ public void ServerPerformInteraction(HandApply interaction)
 	{
 		ServerCombine(interaction.TargetObject.GetComponent<Stackable>());
 	}
+
+	public string Examine(Vector3 worldPos)
+	{
+		string displayName = gameObject.ExpensiveName();
+
+		string str = "This " + displayName + " contains (" + Amount + ") stacks.";
```

Comment:
```suggestion
		return $"This {gameObject.ExpensiveName()} contains {Amount} stacks.";
```
I think it is fine without the variables.

Reply:
added